test/unit/controller/edit_merchant_details_controller/edit_merchant_details_controller_test.js

```diff
@@ -0,0 +1,139 @@
+const sinon = require('sinon')
+const proxyquire = require('proxyquire')
+const chai = require('chai')
+const expect = chai.expect
+const mockResponses = {}
+const mockServiceService = {}
+const random = require('../../../../app/utils/random')
+const merchantDetailsCtrl = proxyquire('../../../../app/controllers/edit_merchant_details_controller', {
+  '../utils/response': mockResponses,
+  '../services/service_service': mockServiceService
+})
+let req, res
+
+describe('when the request body is correct', () => {
```

Comment:
We can combine this test and the ui test as they duplicate similar behaviour. Can pair with you if you want on this.

Reply:
Done! (Will cash in on the pairing offer another time :) )